### PR TITLE
Add support for configuring governor denominator

### PR DESCRIPTION
**Pull request by mrmemes-eth**
The bit packing used to configure both numerator and denominator via the diamond init was a hack to get around the dreaded stack level too deep error from passing too many args to {GovernorDiamondInit_-init}. It works and the interface doesn't seem to suffer from it at the expense of some complexity that remains tucked away pretty well. Still, I'm open to other solutions that will allow initializing the governor in a single transaction.

### CODE REVIEW

File: configs/local-config.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
   "l_votingPeriod": 300,
   "m_votingDelay": 300,
   "n_quorumPercentage": 10,
-  "o_enableGovernaceToken": true,
-  "p_enableMembershipToken": true
-}
```
Comment (desi) [inline configs/local-config.json:18]: Do we need to remove line 15 and add 
`n_quorumNumerator` here like in the example.json?

File: src/utils/GovernorDiamondInit.sol [hunk 2]
```diff
@@ -21,6 +21,13 @@ import {IERC165} from "@diamond/interfaces/IERC165.sol";
 // in order to set state variables in the diamond during deployment or an upgrade
 // More info here: https://eips.ethereum.org/EIPS/eip-2535#diamond-interface
 
+library GDInitHelper {
```
Comment (desi) [inline src/utils/GovernorDiamondInit.sol:24]: I wouldn't have thought to do this this way so very cool.

**Review by desi — APPROVED** [review]
One small question, otherwise nice job.
sith_well_done

File: configs/local-config.json (hunk 1 above)
Comment (mrmemes-eth) [reply configs/local-config.json:18]: Thanks! I have that change locally, but because it's ignored, it didn't pick it up. I'll get that fixed.

**Pull request merged**